tests/juniper_junos/show_system_processes_summary/juniper_junos_show_system_processes_summary.yml

```
@@ -0,0 +1,9 @@
+---
+parsed_sample:
+  - mem_active: "3977M "
+    mem_buffer: "504M "
+    mem_free: "74G "
+    mem_inactive: "14G "
+    mem_wired: "1815M "
+    swap_free: "12G "
+    swap_total: "12G "
```

```suggestion
  - mem_active: "3977M"
    mem_buffer: "504M"
    mem_free: "74G"
    mem_inactive: "14G"
    mem_wired: "1815M"
    swap_free: "12G"
    swap_total: "12G"
```
we should capture trailing space